web/src/components/languages/LanguageSetting.vue

```diff
@@ -0,0 +1,111 @@
+<script lang="ts" setup>
+import { PopoverButton, Popover, PopoverPanel, PopoverOverlay } from "@headlessui/vue"
+import { current, languages, currentAudioLanguage, subtitleLanguage, getNestedLangComputed, setNestedLangMethod } from "@/services/language"
+import { onMounted, ref } from "vue"
+
+const selectedTitle = ref('')
+const isMenuActive = ref(true)
+
+defineProps<{ open: boolean }>()
+
+onMounted(() => {
+	var audLang = localStorage.getItem("audioLanguage") ?? "no"
```

Comment:
Never use `var` in JS. 

Also, localstorage is references multiple places. I suggest moving this somewhere else.